pkg/services/authn/clients/ext_jwt.go

```diff
@@ -129,11 +128,19 @@ func (s *ExtendedJWT) authenticateAsUser(
 		return nil, errExtJWTInvalidSubject.Errorf("unexpected identity: %s", userID.String())
 	}
 
+	// For use in service layer, allow higher privilege
+	allowedKubernetesNamespace := accessTokenClaims.Rest.Namespace
+	if len(s.cfg.StackID) > 0 {
```

Comment:
This is the other thing I would like your review on @kalleep. I think this satisfies my needs for now. I don't want people going about in single tenant context thinking its okay to have an `allowedKubernetesNamespace` value of `*` is alright when an id token is also present.

The MT service layer can escalate up the permissions as it needs.

Reply:
Yes for obo this is what we want to do, nice catch!